openmdao/docs/openmdao_book/features/building_blocks/drivers/doe_driver.ipynb

```diff
@@ -327,6 +327,11 @@
     "\n",
     "import numpy as np\n",
     "\n",
+    "if rank == 0:\n",
+    "    print(\"Found Case\")\n",
+    "else:\n",
+    "    print(\"Found Case\")\n",
+    "    \n",
```

Comment:
this `if` statement seems unnecessary...

Reply:
D'oh! Removed